---
output:
  pdf_document: default
  html_document: default
---
<!-- The following code should appear at the beginning of the first appendix.
After that, all subsequent sections will be turned into appendices. -->

`r if(knitr:::is_latex_output()) '\\Appendices'`

`r if(!knitr:::is_latex_output()) '# (APPENDIX) Appendix {-}'`

# GENETIC STOCK INFORMATION {#app:first-appendix}

Biological information on Yukon Chinook salmon has been collected annually from the fishwheels at White and Sheep rocks (scales; most years from 1985–2008) and the gillnet test fishery at Eagle (tissue samples; 2005–present; \@ref(fig:fig-map-assess)). Samples have typically been taken over the annual upstream adult migration period, with the number of samples taken each day roughly proportional to daily passage (Figure \@ref(fig:fig-gsi-samples)) [@connors_incorporating_2020]. Genetic material has been recovered from 150--300 archived scale samples for most years 1985--2005, and since then 500--1500 tissue samples have been taken per year for genetic stock assignment. These samples have been used to genotype individual fish by microsatellite panel [1985--2016; @beacham2006estimation) or single nucleotide polymorphism (SNP) panel (2017--2019, @beacham2018population and then assigned to one of eight Conservation Units (CUs). These CU groupings differ slightly from the grouping reported in annual Yukon River Panel Joint Technical Committee reports [@JTC2025Report]. This benchmarked baseline consists of 4,713 Chinook salmon sampled at 29 unique spawning locations (or tributaries) throughout the Canadian portion of the Yukon River Basin (Table \@ref(tab:tab-gsi-tribs)). 

To evaluate how accurate genetic assignments were we conducted a simulation analysis on the baseline samples (i.e., samples of known origin) in rubias [@moran2019bayesian]. Simulations indicated that the average simulation accuracy within each CU was greater than 75%, and typically greater than 95%, for all CUs using the microsatellite baseline and similar for the SNP baseline with the exception of the Chandindu, Earn, and Nisling rivers which had poorer assignment accuracies (Table \@ref(tab:tab-gsi-tribs)). Given the variable uncertainty in genetic stock assignments uncovered by these simulations we ensured the run-reconstruction model (Appendix \@ref(app:second-appendix)) fit to these data to derive annual estimates of CU border passage explicitly took uncertainty in genetic stock assignments into account. Genetic samples from the Big Salmon and Teslin (below Teslin lake) tributaries could not reliably differentiated from baseline sampling locations in the Middle Yukon CU; as a result they were grouped with the Middle Yukon CU for the purposes of our genetic analyses. Similarly, genetic samples from the McQuesten River more closely grouped with the Northern CU than with the Stewart CU that they originated from (Table \@ref(tab:tab-gsi-tribs)).These genetic misassignments were subsequently accounted for in the CU scale run-reconstructions (Appendix \@ref(app:second-appendix)).   

\clearpage
(ref:fig-gsi-samples) Daily estimates of Chinook salmon border passage (grey bars) and scales or tissues sampled (red bars) from the run to determine genetic stock identification (GSI). Each bar is relative to the maximum for a given year with GSI samples inverted to allow for visualization of samples relative to border passage. The number of GSI samples in each year is in the bottom right of each panel. Note that scale samples were unavailable for 1988--1990, and 1998. For reference, in a non-leap year 29 June, 29 July and 28 August are equivalent to the 180th, 210th and 240th day of the year, respectively (dashed grey bars).

```{r fig-gsi-samples, fig.cap = "(ref:fig-gsi-samples)"}
knitr::include_graphics(here::here("csasdown/figure/gsi-run-samples.png"))
```
\clearpage

\clearpage

(ref:tab-gsi-tribs) Yukon Chinook salmon populations considered the primary spawning tributaries or location where samples were collected for the genetic baseline (tributary name) total number of baseline genetic samples (Msat(n), SNP(n)), corresponding Conservation Units (CU), CU codes, and probability of correctly assigning a genetic sample for a tributary (collection) back to its known tributary or CU based on simulation analysis. 

```{r tab-gsi-tribs, results = "asis"}

gsi_tribs <- read.csv(here::here("csasdown/data/gsi_tribs.csv"), stringsAsFactors = T)


footnotes <- c("Samples from the Big Salmon river and Teslin river (below Teslin lake) are both assigned to the Middle Yukon CU for the purposes of this analysis because they cannot be reliably differentiated from other baseline sampling locations in the Middle Yukon CU.",
"Holtby and Ciruna (2007)")

gsi_tribs <- gsi_tribs |> 
              dplyr::mutate(Tribname = stringr::str_to_title(gsub("_", " ", Tribname))) |>
              dplyr::mutate(Cucode = gsub("-", " ", Cucode))


names(gsi_tribs) <- c("Tributary name ",	"Msat(n)",	"SNP(n)",	"CU",	"Conservation Unit code$^{2}$",	"Probability to tributary (Msat)",	"Probability to CU (SNP)")


gsi_tribs |>
   dplyr::mutate_all(function(x){gsub("&", "\\\\&", x)}) |> 
   dplyr::mutate_all(function(x){gsub("%", "\\\\%", x)}) |>
   dplyr::mutate_all(function(x){gsub("\\\\n","\n", x)}) |>
   csasdown::csas_table(format = "latex",
                       caption = "(ref:tab-gsi-tribs)", 
                       repeat_header = TRUE) |>
   kableExtra::column_spec(1, width = "3cm")|>
   kableExtra::column_spec(5:7, width = "2cm") 
   #kableExtra::add_footnote(label=footnotes, notation="number") # footnote not working
  
print(footnotes, quote=FALSE)

```


\clearpage



# RUN RECONSTRUCTION MODEL {#app:second-appendix}

We estimated the daily passage of Chinook into Canadian (CDN) waters using a multi-CU run reconstruction (RR) model, which simultaneously accounts for uncertainty in observations and underlying population processes and allows for the inclusion of incomplete datasets. This model was originally described in @connors2022, and was adapted to allow for time-varying catchability and fit to updated data (including genetics and an externally derived border passage index) through 2024. We separately modelled the dynamics of the nine component Conservation Units (CUs), indexed by $i$, in the Yukon Chinook salmon Stock Management Unit (SMU). Annual, CU-specific border passage was the main parameter of interest estimated by the model, though we also estimated parameters for run timing and catchability. We separately modelled time-varying catchability to two fishing “gears” (index by $g$): fishwheels and sonar, which have been used to derive annual aggregate border passage estimates (fishwheel mark-recapture from 1985--2008; sonar from 2005--2024). Model notation and equations are listed in Table \@ref(tab:tab-rr-params) and \@ref(tab:tab-rr-eqns), respectively. While we refer to this model as a run reconstruction, it could also (perhaps more accurately) be referred to as a border passage reconstruction.

(ref:tab-rr-params) Model notation for multi-CU run-reconstruction model. 

```{r tab-rr-params, results = "asis"}
rr.params <- read.csv(here::here("csasdown/data/notations.csv"))
rr.params$Symbol[c(2:5, 7:10, 12:17,19:25)] <- paste0("$",rr.params$Symbol[c(2:5, 7:10, 12:17,19:25)],"$")

rr.params |> 
   dplyr::mutate_all(function(x){gsub("&", "\\\\&", x)}) |> 
   dplyr::mutate_all(function(x){gsub("%", "\\\\%", x)}) |>
   dplyr::mutate_all(function(x){gsub("\n","\\\\\n", x)}) |>
   csasdown::csas_table(format = "latex",
                       caption = "(ref:tab-rr-params)", 
                       repeat_header = TRUE) |>
  kableExtra::column_spec(1, width = "3cm")|>
  kableExtra::column_spec(2, width = "12cm")

```


\begin{landscapepage}
(ref:tab-rr-eqns) Model equations for multi-CU run-reconstruction model. "N.B" stands for negative binomial and "NLL" stands for negative log likelihood.

```{r tab-rr-eqns, results = "asis"}
rr.eqns <- read.csv(here::here("csasdown/data/model-eqns.csv"))

rr.eqns |> 
   dplyr::mutate_all(function(x){gsub("&", "\\\\&", x)}) |> 
   dplyr::mutate_all(function(x){gsub("%", "\\\\%", x)}) |>
   dplyr::mutate_all(function(x){gsub("\n","\\\\n", x)}) |>
   csasdown::csas_table(format = "latex",
                       caption = "(ref:tab-rr-eqns)", 
                       repeat_header = TRUE)|>
  kableExtra::column_spec(1, width = "5cm")|>
  kableExtra::column_spec(1, width = "8cm")


```
\end{landscapepage}

## POPULATION DYNAMICS 

For each population, the daily proportion of salmon entering the model (i.e., crossing the U.S.--Canada border into CDN waters) in a given year is calculated based on a normal probability density function, with mean $\mu_{iy}$ with variance $\sigma^2_i$, which is divided by its sum to ensure that the individual values sum to 1 (Table \@ref(tab:tab-rr-eqns).4). Numbers of salmon arriving daily for each CU/year combination is subsequently calculated as the product of the daily arrival proportions and border passage (Table \@ref(tab:tab-rr-eqns).5). To model interannual variability in mean run timing, we assume that the CU-specific mean dates of arrival in a given year (${\lbrace \mu_{i,y}\rbrace}_{i=1}^9$ or $\mu_{y}$) are a function of run timing in the previous year, i.e.:


\begin{equation}
  \mu_{y}=(\mu_{y-1})^{\epsilon_{y}}
  \label{eq:arrival}
\end{equation}

where $\epsilon_{y}$ is a vector of nine normally distributed process errors with mean 0 and covariance $\sum$. The covariance matrix is constructed as $\sum=DCD$, where $D$ is a diagonal matrix with the variance of $\mu_i$ as the $ii$th element, and $C$ is a symmetric correlation matrix, i.e.:

\begin{equation}
     C_{i,j} = 
      \begin{cases}
         1 &\quad{i = j} \\
         c_{i,j} &\quad{i \neq j} \\
      \end{cases}
  \label{eq:corr_arrival}
\end{equation}

where $c_{i,j}$ is the correlation between arrival timing deviations for CU $i$ and $j$. We constrained all off-diagonal elements of $C$ to have the same value, which assumes all stocks have equal correlation in arrival timing deviations.

## OBSERVATION MODEL AND OBJECTIVE FUNCTION 

Daily border passage counts were predicted by scaling the daily model-predicted passage by a gear-specific catchability factor (Table \@ref(tab:tab-rr-eqns).6). The fishwheel samples an unknown portion of the run, so fishwheel catchability was estimated, and allowed to vary from year to year to account for the potential influence of environmental conditions (e.g., discharge) on fishwheel catchability. While catchability may vary across CUs for a variety of reasons (i.e., bank orientation, run-timing, etc.), the data were not informative enough to estimate CU-specific catchability. In contrast, all passing fish were assumed to be observable by sonar, and so we fixed sonar catchability at 1. Similarly, the predicted border passage index was assumed to represent the total run at he border, so catchability to the border passage index was also fixed at 1 (Table \@ref(tab:tab-rr-eqns).7), and this index was taken from an updated fit of the model described in @connors_estimates_2023. Conservation Unit composition was predicted as the relative proportion of a CU present on each day (Table \@ref(tab:tab-rr-eqns).8).

Daily salmon counts at the border were assumed to arise from negative-binomial ($NB$) distributions (Table \@ref(tab:tab-rr-eqns).13). The ($NB$) distribution, describing the number of successes in a series of Bernoulli trials before $r$ failures occur (with probability of success $p$), is broadly applicable as a model for overdispersed count data. We parameterized the $NB$ distribution in terms of the mean count $\eta$ and the dispersion $\phi$ (Table \@ref(tab:tab-rr-eqns).10-11). The variance of this distribution is $\tau^2=\eta+\eta^2\phi$, hence the $NB$ distribution is equivalent to the Poisson when $\phi=0$. We set $\phi=0$ for sonar counts, as these counts are believed to be relatively accurate. We set $\phi=0.1$ for the fishwheel counts.

Conservation Unit composition data ($x_{igyd}$), which were based on summing the individual genetic assignment probabilities and rescaling to the total sample size for that CU and day thereby accounting for uncertainty in genetic assignments, were fitted using a multinomial likelihood (Table \@ref(tab:tab-rr-eqns).14). For each year/day/gear combination, the observed number of Chinook salmon by CU, $n_1,…,n_9$, was assumed to arise from a multinomial distribution with sample size $n=\sum_in_i$  and probabilities equal to the relative proportions of passage by CU for that year/day. Sample sizes in multinomial distributions for fisheries composition data are typically down-weighted to an “effective” sample size to account for correlations among fish within a given sample. However, we did not down-weight sample sizes as they were already relatively small. We note, however, that by applying a fixed value for the $NB$ dispersion parameter when modelling daily counts as well as to the total run size index likelihood (see below), we are effectively down weighting the stock composition data.

Total border passage indices were assumed to arise from a lognormal distribution with standard deviation $\tau_y$, which was derived from the annual estimates of the CV in the border passage index (Table \@ref(tab:tab-rr-eqns).13). Process errors in arrival timing are assumed to arise from a zero-mean multivariate normal distribution with covariance $\sum$ (Table \@ref(tab:tab-rr-eqns).15).     

In some years prior to the implementation of sonar monitoring, the fishwheel data was not informative enough to estimate border passage for some CUs. To stabilize estimates in these cases, we applied a likelihood penalty on the interannual deviations of log-scale border passage for each CU (Table \@ref(tab:tab-rr-eqns).16). The penalty was set to 1, which is wide enough to have negligible effects in years for which data were sufficiently informative to estimate border passage and was omitted from 2005 onward, after the introduction of sonar assessment of border passage.

## MODEL FITTING AND RECONSTRUCTIONS OF SPAWNER ABUNDANCE 

We fit the run-reconstruction model using Template Model Builder [@kristensen2016tmb]. Model parameters were estimated via maximum likelihood and standard errors of parameters and quantities of interest were calculated using a delta method routine within TMB and were treated as equivalent to standard deviations. 

Fishwheel catchability was estimated to average around 0.02 but varied considerably from year to year (Figure \@ref(fig:fig-RR-catchability)), presumably as a result of different environmental conditions (e.g., flow) and/or fishwheel operation. Interannual deviations in arrival timing were estimated to be weakly correlated among CUs (pairwise correlation, which was assumed to be the same for all CUs, estimated to be equal to 0.15). 

(ref:fig-RR-catchability) Estimated fish wheel catchability, by year. Bars are median estimates +/- 95% confidence intervals.

```{r fig-RR-catchability, fig.cap = "(ref:fig-RR-catchability)"}
knitr::include_graphics(here::here("csasdown/figure/fishwheel-catchability.png"))
```

The model tended to fit the sonar count data well (Figure \@ref(fig:fig-eagle-RR-fits)) and in contrast, model fits to the fishwheel counts for 1985--2007 were more variable, which was expected due to the relatively imprecise nature of the fishwheel data (Figure \@ref(fig:fig-fw-RR-fits)). In years when sonar and fishwheel counts overlapped (2005--2007), the tight correspondence between model estimates and sonar counts resulted in relatively poorer fits to the fishwheel data. 

In the pre-sonar years the model attempted to simultaneously fit to the fishwheel relative abundance indices and the externally derived absolute  border passage index (based on mark-recapture projects) so the model tended to generate estimates of total border passage that were in close alignment with the border passage index (Figure \@ref(fig:fig-fw-RR-fits)). As more reliable abundance data (i.e., sonar based estimates beginning in 2005) became available, the border passage index had essentially no effect on model estimates, and the model fit perfectly to the sonar based estimates of total border passage 2005--2024.

Because the reconstructions of Northern CU border passage included fish from the McQuesten River, we post-hoc estimated the annual contribution of the McQuesten River based on genetic analyses of samples collected at Eagle from 2013-2023 (mean:0.008; SD:0.006). Specifically, for each year we took a random draw from the average McQuesten River contribution to border passage, and a random draw from the border passage estimate for that year, and repeated this 1000 times. We then removed these estimates of McQuesten River border passage from the Northern CU and added them to the reconstructions of Stewart CU border passage. 

The run-reconstruction model estimated border passage and so estimates of spawner abundance were derived by subtracting the inferred harvest of each CU by fisheries that occurred upstream (in Canada) in the mainstem Yukon River from estimated border passage. To do this we assumed all CUs were historically equally vulnerable to Canadian fisheries and calculated spawner abundance as $S_{i,y}=\widehat{R}_{i,y}(1-U_{CDN,y})$ where $\widehat{R}_{i,y}$ is border passage for CU $i$ in year $y$ and $U_{CDN,y}$ is the aggregate harvest rate from Canadian fisheries as estimated by @connors_estimates_2023.

For the Big Salmon CU, which cannot be reliably differentiated with genetics from the Middle Mainstem CU, but which has had a sonar assessment project in operation since 2005, we inferred spawner abundance as a proportion of the reconstructed spawner abundance from the Middle Mainstem CU. This proportion was based on those years where there was data from the sonar assessment project (i.e, 2005 to present with exception of 2022) and the reconstructed spawner abundances explicitly accounted for variability in the proportional Big Salmon CU contribution, and uncertainty in Middle Mainstem spawner abundance, by simulating 1000 estimates of spawner abundance for both CUs by taking random draws from the average proportional contribution of the Big Salmon CU and estimates of Middle Mainstem CU spawner abundance in each year that did not have sonar based Big Salmon assessment. The resulting time series of Big Salmon spawner abundance was then removed for the entire time series from the Middle Mainstem CU.

For the Upper Yukon CU we removed the estimated contribution of hatchery fish based on the count of hatchery fish at the Whitehorse Fishway. 

Resulting estimates of spawner abundance by CU, summed across CU, and as previously estimated for the stock aggregate [@connors_estimates_2023] are provide in Table \@ref(tab:tab-spawn-est). Annual aggregate estimates of spawner abundance from our multi-CU reconstructions and previous   reconstructions [@connors_estimates_2023] were, on average, within 6% of each other (~9% in years prior to Eagle sonar, and ~3% in years thereafter).  

\clearpage
(ref:fig-eagle-RR-fits) Fits of the run-reconstruction model (lines) to daily sonar counts (circles) at Eagle, near the U.S. --Canada border.

```{r fig-eagle-RR-fits, fig.cap = "(ref:fig-eagle-RR-fits)"}
knitr::include_graphics(here::here("csasdown/figure/indexFitsg1.png"))
```

\clearpage
(ref:fig-fw-RR-fits) Fits of the run-reconstruction model (lines) to daily fishwheel counts (circles) at White and Sheep Rocks, near the U.S. --Canada border.

```{r fig-fw-RR-fits, fig.cap = "(ref:fig-fw-RR-fits)"}
knitr::include_graphics(here::here("csasdown/figure/indexFitsg2.png"))
```

\clearpage

(ref:fig-run-RR-fits) Total border passage estimates based on the run-reconstruction model (grey circles with bars indicating the central 95% confidence interval), mark-recapture based estimates of run size from fish sampled from fishwheels after accounting for selectivity (open square), annual sum of daily fishwheel counts (scaled by estimated catchability, open green triangle) and the Eagle sonar (red circle).

```{r fig-run-RR-fits, fig.cap = "(ref:fig-run-RR-fits)"}
knitr::include_graphics(here::here("csasdown/figure/CU-RR-fits.png"))
```
\clearpage
(ref:tab-spawn-est) Median estimates of spawner abundance by CU, summed across CU ("Aggregate"), and as previously estimated for the stock aggregate (Connors et al. 2022). 
```{r tab-spawn-est, results='asis'}
spawntable <- read.csv(here::here("analysis/data/generated/spawn-table.csv"))
names(spawntable)[1] <- "Conservation Unit or stock aggregate"

spawntable |> 
  dplyr::mutate_all(function(x){gsub("\\\\.", " ", x)}) |>
  dplyr::mutate_all(function(x){gsub("\\\\n","\n", x)}) |>
  dplyr::mutate_all(function(x){gsub("%", "\\\\%", x)}) |>
  csasdown::csas_table(format = "latex",
                       escape = FALSE,
                     caption = "(ref:tab-spawn-est)", 
                     repeat_header = TRUE,
                     linesep = "\\addlinespace") |>
    kableExtra::column_spec(1, width = "7cm")|>
    kableExtra::column_spec(2, width = "2cm")|>
    kableExtra::column_spec(3, width = "2cm") 
```

\clearpage

# SPAWNER RECRUITMENT MODELS  {#app:third-appendix}

To characterize Yukon Chinook salmon CU spawner-recruitment relationships we fit individual age-structured state-space spawner-recruitment models [@fleischman-etal-2013] to all available data for each CU (i.e., time series of estimated catch, spawner abundance and age composition).

These sources of information, along with measures of observation variance, were externally derived from the multi-CU run-reconstructions described in Appendix \@ref(app:first-appendix) for spawner abundance and published estimates for catch and age composition [@connors_estimates_2023], updated through 2024.

An alternative to the sequential modelling approach we took would have been to integrate the run-reconstructions and spawner-recruitment analyses into a single model. However, we note that previous research has found that as long as the spawner-recruitment model carries forward uncertainty in input estimates from the run-reconstruction models, sequential analyses like those that we conducted (which still do not account for parameter covariance among models) produce very similar estimates of abundance, population dynamics parameters, and management reference points, both in terms of point estimates and uncertainty [@staton-etal-2017].

These models could have also been fit in a multi-population framework [@connors_incorporating_2020; @staton-etal-2020]. However, these integrated models tend to yield very high estimates of recruitment variability and negative estimates of recruitment autocorrelation which appear to be spuriously too high  and low, respectively, and which could lead to biases in other model parameters in manners that are difficult to predict [@staton-etal-2020].

Because the same model was fitted to each CU, all variables in the presentation of the state-space model actually have a $j$ subscript appended to them to represent individual CUs. We omit this for the presentation here, both to simplify it and to emphasize that this approach fits to data from each CU separately.

## PROCESS MODEL

The process model is intended to represent the true population dynamics (i.e., free of measurement error), and this component of our state-space spawner-recruitment model specifies productivity, density-dependence, and age-at-return by cohort (i.e. brood year, $y$).

Recruitment abundances of adult Chinook salmon (i.e., the total number of adults from a given brood year that will ever return, $R_y$) were treated as unobserved states and modeled as a function of spawner abundance in year $y$ ($S_y$) assuming a @ricker-1954 spawner-recruitment relationship with serially auto-correlated log-normal process variation:

\begin{equation}
  \ln(R_y) = \ln(S_y) + \ln(\alpha) - \beta S_y + v_y
  \label{eq:AR1-ricker}
\end{equation}

where $\alpha$ is productivity (intrinsic rate of growth), $\beta$ is the magnitude of within brood year density-dependent effects, and $v_y$ reflects inter-annual variation in survival from egg to adulthood, which we term "recruitment anomalies".

This variation was assumed to follow a lag-1 autoregressive process (correlation coefficient denoted by $\phi$) over time:

\begin{equation}
  \begin{aligned}
  v_y &= \phi v_{y-1} + \varepsilon_y \\
  \varepsilon_y &\sim \mathcal{N}(0, \sigma_R)
  \end{aligned}
  \label{eq:eq-residuals}
\end{equation}

where $\varepsilon_y$ reflects the portion of the recruitment anomaly $v_y$ that is temporally independent (i.e., white noise).

The first seven years of recruitment states were not linked to observations of spawner abundance in the spawner-recruitment relationship (Equ. \@ref(eq:AR1-ricker)) and were modeled as random draws from a log-normal distribution with mean $\ln(R_0)$ and variance $\sigma_R^2$.

Rather than estimating $\ln(R_0)$ as a free parameter as in @fleischman-etal-2013, we choose to follow @staton-etal-2020 and inform its value using the expected recruitment under equilibrium unfished conditions $\ln(\alpha)/\beta$.

The number of Chinook salmon that returned to spawn in year $y$ at age $a$ ($a \in$ 4:7) was the product of the total recruitment produced by spawning that occurred in brood year $y-a$ and the proportion from brood year $y-a$ that returned at age $a$:

\begin{equation}
  N_{y,a} = R_{y-a} p_{y-a,a}
  \label{eq:returnage}
\end{equation}

where $p_{y,a}$ is the probability that a fish spawned in brood year $y$ will return at age $a$. Because we lacked empirical data on life stage-specific abundances (e.g., smolts, fish abundance of different ages/brood years in the ocean) from which to directly estimate maturation rates and life-stage specific mortality, the sole purpose of this probability of return-at-age model (Equ. \@ref(eq:returnage)-\@ref(eq:returnagevar)) is to apportion the abundance of total adult recruitment to the correct age and year of return.

We modeled brood year variation in age at return as Dirichlet random vectors drawn from a common hyperdistribution characterized by a mean age-at-maturity probability vector ($\pi$) and an inverse dispersion parameter ($1/D^2$):

\begin{equation}
  p_{y,a}  \stackrel{\mathrm{iid}}{\sim}  \mathcal{D}(\pi(\frac{1}{D^2}))
  \label{eq:returnagevar}
\end{equation}

The total run in year $y$ was made up of recruitments from multiple brood years as shown in Equ. \@ref(eq:returnage) and was calculated as the sum of the age-specific run abundances:

\begin{equation}
  N_{y} = \sum_{a=4}^{7} N_{y,a}
  \label{eq:totalreturn}
\end{equation}

Harvest in a given year ($H_y$) was modeled as the product of total run size and the harvest rate ($U_y$) experienced that year:

\begin{equation}
  H_y = N_y U_y
  \label{eq:harvest}
\end{equation}

and spawner abundance ($S_y$) was modeled as the portion of $N_y$ remaining after harvest $H_y$:

\begin{equation}
  S_y = N_y (1 - U_y)
  \label{eq:spawners}
\end{equation}

It is this spawner abundance $S_y$ (which is itself made up of fish of multiple ages from multiple brood years) that is used in Equ. \@ref(eq:AR1-ricker) to produce the recruitment abundance from spawning in brood year $y$.


## OBSERVATION MODEL

We assumed a coefficient of variation (CV) for spawner observations that was based on the annual CV in estimated spawner abundances derived from the run reconstruction model (Appendix \@ref(app:first-appendix)).

Observed spawner abundance ($S_{obs,y}$) was therefore assumed to be log-normally distributed with the CVs converted to log-normal variance following @forbes-etal-2011:

\begin{equation}
  \sigma^2_{o,y} = \ln\left(\mathrm{CV}_y^2 + 1\right)
  \label{eq:get-sigm}
\end{equation}

Direct estimates of harvest by CU do not exist because most harvest occurs in mixed-stock fisheries in the mainstem of the Yukon River, and this harvest has never been apportioned back to CUs. Harvest observations ($H_{obs,y}$) were therefore approximated based on the stock aggregate harvest rate [$U_{obs,y}$, @connors_estimates_2023] and CU specific spawner abundance:

\begin{equation}
  H_{obs,y} = S_{obs,y}(1-U_{obs,y})^{-1}U_{obs,y}
  \label{eq:obs-harvest}
\end{equation}

and were assumed to have a CV equal to that of the joint variability in spawner abundance and aggregrate harvest rate estimates which was log-normally distributed and converted to log-normal variance as per Equ. \@ref(eq:get-sigm).

Age composition by return year at the CU scale was available from fish passing from the U.S. into Canada most years from 2008 to present, but samples sizes are small because not all fish that were aged have corresponding genetic stock identification information, and vice-versa. In earlier years (i.e., pre 2008) it was often difficult to match individual scale samples used for ageing with scales that had genetic material extracted making any inference on CU age composition unreliable. In addition, while these sources of data provide insights into age composition of fish prior to spawning they may not be representative of total return age composition because harvest can be size selective and these samples are taken after the bulk of harvest occurs. We therefore made the simplifying assumption that estimates of age composition by return year, taken from an integrated run-reconstruction and spawner-recruitment model that explicitly takes age composition in harvest and spawners into account [@connors_estimates_2023], were a reasonable proxy for CU specific age compositions. We felt this was a reasonable assumption because CU specific border age composition from 2008 to present tended to be very similar among CUs (Figure \@ref(fig:fig-age-compare)). 

These age composition by return year estimates were assumed to be observed with multinomial sampling error, where uncertainty in age proportions in a given year was generated by specifying an "effective sample size" (ESS) of 50 pre-2007 and 100 thereafter. These ESSs are lower than the likely observed sample sizes to account for the non-independence of observation within a given sampling event [@maunder_review_2011] and were chosen to reflect the reduced confidence in age composition data prior to the Eagle test fishery sampling program which was standardized beginning in 2007.

## TIME-VARYING PRODUCTIVITY

Given widespread evidence for changes in salmon productivity over time [@peterman_widespread_2012; @dorner2018spatial; @malick2016regional], and interest in understanding the evidence for changing productivity in Yukon Chinook salmon, we also fit a version of the model described in Equ. \@ref(eq:AR1-ricker)-\@ref(eq:obs-harvest) with time-varying intrinsic productivity (Table \@ref(tab:tab-tva-pars)). Specifically, we allowed the $\alpha$ parameter to evolve through time as a random walk, yielding time-varying estimates of productivity: 

\begin{equation}
  \begin{aligned}
  \alpha_y &= \alpha_{y-1} + \varepsilon_y \\
  \varepsilon_y &\sim \mathcal{N}(0, \sigma_\alpha)
  \end{aligned}
  \label{eq:tv-alpha}
\end{equation}

where $\sigma_\alpha$ is the magnitude of process variation in productivity and where recruitment anomalies were no longer modeled as being auto-correlated but all other parameters in equation \@ref(eq:AR1-ricker) otherwise remained the same.


(ref:tab-tva-pars) Estimates of key parameters from time-varying $alpha$  spawner-recruitment models by Conservation Unit along with two measures of parameter estimability and model convergence. Estimates of $\beta$ are on the scale of 10,000 fish for ease of summarizing parameters in the table. For $\alpha$ and $\sigma_\alpha$, which are time-varying parameters, values are averages of the posterior median estimates for each year along with the standard deviation among years.
```{r tab-tva-pars, results = "asis"}

pars <- read.csv(here("analysis/data/generated/TVA_par_summary.csv"))

CU_name_lookup <- data.frame(CU = c("NorthernYukonR.andtribs.", 
              "Whiteandtribs.", "Stewart",  
              "MiddleYukonR.andtribs.","Pelly",
              "Nordenskiold", "Big.Salmon", 
              "UpperYukonR.","YukonR.Teslinheadwaters"), 
                CU_pretty = c("Northern Yukon R. and tribs.", 
                "White and tribs.", "Stewart",  
                "Middle Yukon R. and tribs.","Pelly",
                "Nordenskiold", "Big Salmon", 
                "Upper Yukon R.","Yukon R. Teslin Headwaters"))

alpha <- pars |> filter(grepl("^ln_alpha", X)) |> 
  group_by(CU) |> 
  summarize(mean = mean(X50.), sd=sd(X50.), n_eff=mean(n_eff), Rhat=mean(Rhat)) |>
  mutate(mean = round(mean, 2), sd=round(sd, 2)) |>
  mutate(par = "alpha", value = paste(mean, "(",sd,")")) |>
  select(-c(mean,sd))
  

alpha_dev <- pars |> filter(grepl("^alpha_dev", X)) |> 
  group_by(CU) |> 
  summarize(mean = mean(X50.), sd=sd(X50.), n_eff=mean(n_eff), Rhat=mean(Rhat)) |>
  mutate(mean = round(mean, 2), sd=round(sd, 2)) |>
  mutate(par = "alpha_dev", value = paste(mean, "(",sd,")")) |>
  select(-c(mean,sd))

pars <- pars |> filter(grepl("^beta", X)) |> 
  rename(value = X50.) |> mutate(par = "beta") |>
  mutate(value = formatC(value*10000, format='f', digits=2)) |>
  select(CU, n_eff, Rhat, par, value) |>
  rbind(alpha_dev) |> rbind(alpha) |>
  left_join(CU_name_lookup, by="CU") |>
  select(CU_pretty, par, value, n_eff, Rhat) |>
  mutate(n_eff = round(n_eff, 0), Rhat = formatC(Rhat, format='f', digits=3))


names(pars) <- c("Conservation Unit",
                  "Parameter",
                  "Mean (SD) or Posterior Median",
                  "Effective Sample Size ($N_{\\text{eff}}$)",
                  "Chain Mixing ($\\widehat{R}$)")

pars$Parameter[pars$Parameter == "alpha"]<- "$\\alpha$"
pars$Parameter[pars$Parameter == "beta"]<- "$\\beta$"
pars$Parameter[pars$Parameter == "alpha_dev"]<- "$\\sigma_\\alpha$"

pars |> 
  dplyr::arrange(`Conservation Unit`, `Parameter`) |>
  dplyr::mutate_all(function(x){gsub("&", "\\\\&", x)}) |> 
  dplyr::mutate_all(function(x){gsub("%", "\\\\%", x)}) |>
  dplyr::mutate_all(function(x){gsub("\n","\\\\n", x)}) |>
  csasdown::csas_table(format = "latex",
                     caption = "(ref:tab-tva-pars)", 
                     repeat_header = TRUE,
                     align = "llc")  |>
  kableExtra::column_spec(3, width = "2.5cm")|>
  kableExtra::column_spec(4, width = "1.5cm")|>
  kableExtra::column_spec(5, width = "1.5cm") |>
  kableExtra::collapse_rows(columns=1, latex_hline = "major")

```


## DEMOGRAPHIC CHANGE
The spawner-recruitment models described in the previous section assumed that demographic attributes of spawning escapement have not changed over time such that reproductive output is homogeneous among individuals and has been static over the past 40 years. However, changes in age, sex, and length-at-age of returning Chinook salmon, including in the Yukon [@lewis_changes_2015; @ohlberger_demographic_2018; @ohlberger_reproductive_2020], have been widely observed, suggesting concurrent declines in per capita reproductive output as smaller females carry disproportionately fewer eggs of a smaller mass compared to larger females (i.e., fecundity and total egg mass scale exponentially with female size) [@ohlberger_reproductive_2020]. While changes in age, sex, and length-at-age (aka “escapement quality”) are not typically considered when deriving spawner-recruitment based reference points and setting escapement goals, previous research on Chinook salmon in the Yukon and Kuskokwim rivers of western Alaska suggests that failing to account for declines in escapement quality may result in underestimating the escapement needed to maximize long-term sustainable yield or recruitment [@staton_incorporating_2021;@connors_estimates_2023]. In addition, simulation analyses suggest that accounting for the consequences of declining "escapement quality" in fishery management (via reference points) can reduce conservation risks relative to assessment frameworks that do not take demographic change into account [@ohlberger2025accounting].  

To evaluate the potential consequences of changes in escapement quality over time (which we operationalized as total egg mass) we translated time trends in sex ratios, age composition, and size-at-age to annual estimates of total egg mass production. We chose to focus on total egg mass instead of eggs because it it tightly correlated with egg energy content [@ohlberger_reproductive_2020] and may be a better proxy for female reproductive output than egg number. Large females produce not only more but also larger eggs compared to small females [@ohlberger_reproductive_2020]. 

Estimates of annual age and sex composition, and mean length were derived from records of individual fish sampled by fishwheels (pre-2007), and more recently by multi-mesh size gill net test fishery, near the U.S.--Canada border (ADF&G AYK Database Management System). These data were corrected for the known size selectivity of fishwheels using the length selectivity method described in @Hamazaki2018BorderAgeComp .The allometric relationship between female Chinook salmon size and total egg mass that we assumed was based on 140 samples of female fish with paired measurements of length (mid-eye to tail fork; METF), egg count and egg (ovary) mass, based on collections in the gill net test fishery between 2008 and 2010 [@ohlberger_reproductive_2020].  

With this alternative measure of reproductive potential we then refit our our spawner-recruitment models, replacing the spawner abundance term ($S_{y}$) in Equ. \@ref(eq:AR1-ricker) with:  

\begin{equation} 
  Z_{m,y} = \sum_a S_{y} \cdot q_{y,a} \cdot w_{em,y,a} 
  (\#eq:reprometric)
\end{equation}

where $Z_{m,y}$ is the year-specific total egg mass $em$, $S_{y}$ is the year specific total spawner abundance, $q_{y,a}$is the year- and age-specific proportion females, and $w_{em,y,a}$ is the average total egg mass ($em$) per individual in each year ($y$) and age ($a$) (Table \@ref(tab:tab-emr-pars)).

This model makes several key assumptions including that total reproductive output is limited by females and that male abundance is always sufficient to fertilize all eggs [@staton_incorporating_2021]. We believe these were reasonable assumptions across the range of spawner abundances that have been observed, and at the scale at which we are modelling spawner-recruitment dynamics, though at very low abundances and in small populations, male abundance will also presumably limit reproductive output. The model also assumes that egg mass suitably indexes within-stock density effects whereas total spawners (males and females) contribute to density effects in the base model formulation. For these reasons, plus necessary assumption that border age, size, and sex composition is a reasonable proxy for CU scale demographic characteristics, we consider these demographic models exploratory and encourage greater consideration of consequences of potentially violating these assumptions in future analyses.


(ref:tab-emr-pars) Estimates of key egg mass-recruitment relationship parameters by Conservation Unit along with two measures of parameter estimability and model convergence. Estimates of $\beta$ are on the scale of 10,000 fish for ease of summarizing parameters in the table.
```{r tab-emr-pars, results="asis"}
pars <- read.csv(here("analysis/data/generated/demographic_par_summary.csv"))

CU_name_lookup <- data.frame(CU = c("NorthernYukonR.andtribs.", 
              "Whiteandtribs.", "Stewart",  
              "MiddleYukonR.andtribs.","Pelly",
              "Nordenskiold", "Big.Salmon", 
              "UpperYukonR.","YukonR.Teslinheadwaters"), 
                CU_pretty = c("Northern Yukon R. and tribs.", 
                "White and tribs.", "Stewart",  
                "Middle Yukon R. and tribs.","Pelly",
                "Nordenskiold", "Big Salmon", 
                "Upper Yukon R.","Yukon R. Teslin Headwaters"))

alpha <- pars |> filter(grepl("^lnalpha", X)) |> 
  mutate_at(2:7, function(x){signif(exp(x), 2)}) |>
  mutate(n_eff = round(n_eff, 0), Rhat=formatC(Rhat, format='f', digits=3)) |>
  mutate(value = paste(X50., "(", X2.5. , ",", X97.5., ")")) |>
  select(CU, X, value, n_eff, Rhat)

beta <- pars |> filter(grepl("^beta", X)) |> 
  mutate_at(2:7, function(x){signif(x*10000, 2)}) |>
  mutate(n_eff = round(n_eff, 0), Rhat=formatC(Rhat, format='f', digits=3)) |>
  mutate(value = paste(X50., "(", X2.5. , ",", X97.5., ")")) |>
  select(CU, X, value, n_eff, Rhat)

phi <- pars |> filter(grepl("^phi", X)) |> 
  mutate_at(2:7, function(x){signif(x, 2)}) |>
  mutate(n_eff = round(n_eff, 0), Rhat=formatC(Rhat, format='f', digits=3)) |>
  mutate(value = paste(X50., "(", X2.5. , ",", X97.5., ")")) |>
  select(CU, X, value, n_eff, Rhat)

pars <- bind_rows(list(alpha=alpha, beta=beta, phi=phi), .id="par") |>
  left_join(CU_name_lookup, by="CU") |>
  select(CU_pretty, par, value, n_eff, Rhat)


names(pars) <- c("Conservation Unit",
                  "Parameter",
                  "Median 95$^{\\text{th}}$ percentiles)",
                  "Effective Sample Size ($N_{\\text{eff}}$)",
                  "Chain Mixing ($\\widehat{R}$)")

pars$Parameter[pars$Parameter == "alpha"]<- "$\\alpha$"
pars$Parameter[pars$Parameter == "beta"]<- "$\\beta$"
pars$Parameter[pars$Parameter == "phi"]<- "$\\phi$"

pars |> 
  dplyr::arrange(`Conservation Unit`, `Parameter`) |>
  dplyr::mutate_all(function(x){gsub("&", "\\\\&", x)}) |> 
  dplyr::mutate_all(function(x){gsub("%", "\\\\%", x)}) |>
  dplyr::mutate_all(function(x){gsub("\\\\n","\n", x)}) |>
  csasdown::csas_table(format = "latex",
                     caption = "(ref:tab-emr-pars)", 
                     repeat_header = TRUE,
                     align = "llc") |>
  kableExtra::column_spec(3, width = "3.5cm")|>
  kableExtra::column_spec(4, width = "1cm")|>
  kableExtra::column_spec(5, width = "1.5cm") |>
  kableExtra::collapse_rows(columns=1, latex_hline = "major")

```


## MODEL FITTING AND DIAGNOSTICS 

We fit the spawner-recruitment models in a Bayesian estimation framework with Stan [@carpenter_stan_2017; @standevelopmentteamRstanInterfaceStan2023], which implements the No-U-Turn Hamiltonian Markov chain Monte Carlo (MCMC) algorithm [@hoffman2014] for Bayesian statistical inference to generate a joint posterior probability distribution of all unknowns in the model. Priors were generally uninformative or weakly informative and are summarized in Table \@ref(tab:tab-priors).

We sampled from 4 chains with 3000 iterations each and discarded the first 1000 as warm-up. 
We assessed chain convergence visually via trace plots and by ensuring that $\hat{R}$ [potential scale reduction factor, @vehtari2021rank] was generally less than 1.1 and that the effective sample size was greater than 400, or 10% of the iterations. Posterior predictive checks were also used to make sure that the model returned data similar to the data used to fit parameters. A summary of model diagnostics is available online [here](https://pacific-salmon-assess.github.io/yukon-CK-ResDoc/analysis/R/SR-diagnostics.html). Most models had leading parameters that were well estimated but the stationary spawner-recruitment model fit to data from the Teslin CU had relatively low effective sample size suggesting posterior samples were not well mixed with high autocorrelation. This was likely a result of the strong temporal pattern in recruitment residuals as evidenced by high effective samples sizes for the model that allowed for variation in intrinsic productivity over time.     

\clearpage
(ref:tab-priors) Prior probability distributions for leading spawner-recruitment model parameters. Note that the egg mass and recruitment model shared the same priors except for $ln(\alpha)$ which was given a prior of $\sim N(-6,2)$ with bounds $[-10,5]$ because the parameter was on the scale of grams and not individual fish.
```{r tab-priors, results = "asis"}
priors.df <- data.frame(Parameter = c("$ln(\\alpha)$", 
                                      "$\\beta$", 
                                      "$\\phi$", 
                                      "$\\sigma_R$", 
                                      "$ln(R_0)$", 
                                      "$\\sigma_{R_0}$",
                                      "$\\pi$",
                                      "$D$"), 
                        Prior = c("$\\sim N(1,2)$", 
                                  "$\\sim N(0,1)$", 
                                  "$\\sim U(-1,1)$", 
                                  "$\\sim N(0,2)$", 
                                  "$\\sim N(0,20)$", 
                                  "$\\sim Inv-Gamma(2,1)$",
                                  "$\\sim Dir(0.25,0.25,0.25,0.25)$",
                                  "$\\sim Beta(1,1)$"),
                        Bounds= c("$[0,\\inf]$",
                                  "$[0,\\inf]$", 
                                  "$[-1,1]$", 
                                  "$[0,\\inf]$",
                                  "$[0,\\inf]$", 
                                  "$[0,\\inf]$",
                                  "",
                                  ""),
                        Description = c("Natural log of intrinsic rate of growth.",
"Magnitude of within brood-year density-dependence",
"Lag-one correlation in interannual variation in survival.", 
"White noise process standard deviation in survival.", 
"Natural log of unobserved recruitment in the first year of process model.",
"Standard deviation in unobserved recruitment in the first year of process model.",
"Mean maturation-at-age probability for ages 4:7.",
"Dispersion parameter that governs variability in maturation-at-age probabilities across cohorts."))

priors.df |>
   dplyr::mutate_all(function(x){gsub("&", "\\\\&", x)}) |> 
   dplyr::mutate_all(function(x){gsub("%", "\\\\%", x)}) |>
   dplyr::mutate_all(function(x){gsub("\\\\n","\n", x)}) |>
  csasdown::csas_table(format = "latex",
                     caption = "(ref:tab-priors)", 
                     repeat_header = TRUE) |>
    kableExtra::column_spec(4, width = "5cm") |>
    kableExtra::column_spec(2, width = "6cm")
```

## BIOLOGICAL BENCHMARKS 
We calculated biological benchmarks points for each MCMC sample and then summarized them across MCMC samples. The spawning abundance expected to maximize sustainable yield over the long-term under equilibrium conditions, $S_{MSY}$ was derived as:  

\begin{equation}
  S_{MSY} = 1 - W(e^{1-ln(\alpha)})/\beta
  \label{eq:get-Smsy}
\end{equation}

where $W$ is the Lambert function [@scheuerellExplicitSolutionCalculating2016], and $\alpha$ and $\beta$ are intrinsic productivity and the magnitude of within stock density dependence, respectively. We chose to apply this exact solution for $S_{MSY}$ instead of the commonly applied @hilborn1985simplified approximation because the approximation only holds for $0 <ln(\alpha) \leq3$ such that infrequent, but large, posterior samples of $\alpha$ can result in biased estimates of the posterior distribution of $S_{MSY}$. 
 

The spawner abundance expected to result in the stock rebuilding to $S_{MSY}$ in one generation in the absence of fishing [$S_{gen}$, @holtEvaluationBenchmarksConservation2009], often used as a lower biological benchmarks for CUs under the Wild Salmon Policy, was solved numerically according to: 

\begin{equation}
  S_{MSY} = S_{gen}\alpha e^{-\beta S_{gen}}
  \label{eq:get-Sgen}
\end{equation}

The spawner abundance expected to maximize recruitment over the long-term under equilibrium conditions ($S_{MSR}$, also commonly referred to as $S_{MAX}$) was estimated as: 

\begin{equation}
  S_{MSR} = \frac{1}{\beta}
  \label{eq:get-Smsr}
\end{equation}

Equilibrium spawner abundance ($S_{eq}$), where recruitment exactly replaces spawners, was estimated as:  

\begin{equation}
  S_{eq} = ln(\alpha)/\beta
  \label{eq:get-Seq}
\end{equation}

The harvest rate expected to lead to maximum sustainable yield, $U_{MSY}$, was derived according to the solution proposed by @scheuerellExplicitSolutionCalculating2016 as:  

\begin{equation}
  U_{MSY} = 1 - W(e^{1-ln(\alpha)})
  \label{eq:get-Umsy}
\end{equation}

To derive biological benchmarks from the egg mass-recruitment models we used the yield-per-recruit algorithm described in @staton_incorporating_2021. Specifically, we calculated reproductive output-per-recruit ($z_{PR}$) under fishing mortality $F$ as a function of the age and sex class ($i$) - specific reproductive output ($z_{i,j}$) in a given time block ($j$) weighted by the probability of returning by age and sex in that time block ($\omega_{i,j}$) and the probability of escaping harvest ($1-U_{i,j}$):

\begin{equation}
  z_{PR,j} = \sum_{i}(1-U_{i,j})z_{i,j}\omega_{i,j}
  \label{eq:get-YPR}
\end{equation}

where $U_{i,j} = 1-exp(-F)$ and time block $j$ was either the first generation (1985-1990), all years (1985-2024), or the last generation (2019-2024) of observed spawners. We then could calculate equilibrium recruitment as:

\begin{equation}
  R_{eq,j} = \frac{log(\alpha z_{PR,j})}{\beta z_{PR,j}}
  \label{eq:get-eqR}
\end{equation}

Once the scale of the equilibrium population fished at $F_{max}$ was obtained from Eqn. \@ref(eq:get-eqR), we calculated the age- or sex-structured total return, harvest, and spawning escapement:

\begin{equation}
  \begin{aligned}
  N_{eq,i,j}=R_{eq,j}\omega_{i,j} \\
  H_{eq,i,j}=N_{eq,j}U_{i,j} \\
  S_{eq,i,j}=N_{eq,j}(1-U_{i,j}) \\
  \end{aligned}
  \label{eq:get-eqNHS}
\end{equation}

and calculated totals for harvest and spawning escapement by summing over ages and sexes. 

We then treated $H_{eq,i,j}$ or $R_{eq,i,j}$ as the objective value to maximize via iterative numerical search on the quantity $F_{max}$ to estimate demographically based biological benchmarks $S_{MSY,j}$ or $S_{MSR,j}$ for a given time period, respectively.
```{r deps}
source("../analysis/R/load.R")
```


(ref:fig-bench-compare) Comparison of alternative (a) lower and (b) upper biological benchmarks as well as (c) estimates of maximum returns ($S_{MSR}$) or equilibrium population size ($S_{EQ}$) across CUs. The numeric value in legends refers to a percentage of the benchmark and those labelled "egg-mass" are derived from analyses that account for demographic change and are based on demographic characteristics observed over the most recent generation (2019-2024). Bars are medians and whiskers are upper 95th percentiles.

```{r fig-bench-compare, fig.cap = "(ref:fig-bench-compare)", out.width = "70%"}
knitr::include_graphics(here::here("csasdown/figure/bench-compare.png"))
```
# SENSITIVITY ANALYSES {#app:sixth-appendix}
## AGE COMPOSITION
Our analysis of CU spawner recruitment dynamics made the simplifying assumption that estimates of age composition by return year, taken from an stock aggregate run-reconstruction and spawner-recruitment model that explicitly takes age composition in harvest and spawners into account [@connors_estimates_2023], were a reasonable proxy for CU specific age compositions. We made this assumption because age composition information at the CU scale was unavailable because individual samples taken from fish for genetics at the border, and samples taken for ageing, could not be reliably linked in many years. 

However, those years where age and genetic samples could be linked allow for a qualitative examination of the degree to which this simplifying assumption might be violated. For those years where there were 1000 or more linked genetic and age samples we compared age proportions by CU (Figure \@ref(fig:fig-age-compare)). These comparisons illustrate that while age composition differed among some CUs in some years, overall it was broadly similar across CUs within years and there was no evidence of consistent over/under representation of age classes unique to a given CU. The exception to this was the Nordenskiold CU which is some years exhibited very different age composition, though we caution against overinterpreting this pattern because it is the smallest CU and often had 10 or less samples in a given year.  

(ref:fig-age-compare) Age composition in Chinook salmon sampled at the Alaska-Yukon Territory border, by Conservation Unit.  
```{r fig-age-compare, fig.cap = "(ref:fig-age-compare)"}
knitr::include_graphics(here::here("csasdown/figure/age-cu-by-yrs.png"))
```

In addition to the comparison of stock aggregate and CU scale border age composition we also compared stock aggregate age composition to Blind Creek (Pelly CU) age composition based on samples taken from fish on the spawning grounds. This comparison allowed us to qualitatively evaluate the degree to which aggregate age composition estimates at the border were a reasonably proxy for age composition on the spawning grounds. This comparison highlights that for those years where a comparison could be made there was reasonable agreement between the two sources of data on age composition.      

(ref:fig-blind-age-compare) Age composition of Chinook salmon sampled at the the Blind Creek weir (in Pelly CU) and those sampled at the Alaska-Yukon Territory border.  
```{r fig-blind-age-compare, fig.cap = "(ref:fig-blind-age-compare)"}
knitr::include_graphics(here::here("csasdown/figure/blind-age-by-yrs.png"))
```
## VULNERABILITY TO HARVEST
Our reconstructions of CU scale harvests, and subsequent population dynamics modelling and estimation of biological benchmarks, relied on the assumption that all CUs have historically experienced the same harvest rates. While this was a necessary assumption due to a lack of routine genetic assignment of fish sampled in fisheries to CUs, a multi-year study of genetic composition of fish harvested in the Alaskan portion of the Yukon in the mid to late 2000s allowed us to qualitatively examine this assumption. From 2005 to 2010 Chinook salmon were sampled in commercial and subsistence fisheries in the Alaskan portion of the Yukon River and estimates of genetic stock composition were derived for stock groupings that consisted of either individual or groups of CUs [e.g., @DeCovichHarvStockComp2011]. For each year, and river district, we calculated harvest by CU group as the product of total Canadian-origin Yukon Chinook harvest and CU group composition (\@ref(eq:obs-harvest)). We then summed these estimates across districts to calculate the total CU group composition for Canadian-origin Yukon Chinook harvested in Alaska for a given year, and compared it to estimates of CU group composition at the Alaska-Yukon Territory border based on the border passage reconstructions described in this document (Appendix \@ref(app:second-appendix)).

For these few years where genetic sampling of harvest in Alaska allowed for a comparison of CU composition in both border passage and harvest there was no evidence of consistent over/under-representation of individual CUs in Alaskan harvest, though in some years for some CU groups there were qualitative differences in representation (Figure \@ref(fig:fig-CU-harv-compare)). This suggests that overall, across years with data, CUs were generally equally vulnerable to harvest in Alaskan fisheries.

(ref:fig-CU-harv-compare) Conservation Unit group composition of Chinook salmon sampled in commercial and subsistence fisheries in Alaska ("Harvest") and fish sampled in test fisheries at the Alaska-Yukon Territory border ("Border"). The "Border" CU grouping includes the Northern Yukon and White and tributaries CUs, the "Pelly" grouping includes the Pelly and Stewart CUs, the "Carmacks" grouping includes the Nordenskiold, Middle Yukon, Big Salmon and Teslin CUs, and the "Takhini" grouping includes the Upper Yukon CU.  

```{r fig-CU-harv-compare, fig.cap = "(ref:fig-CU-harv-compare)"}
knitr::include_graphics(here::here("csasdown/figure/harbest_vs_border_composition.png"))
```

# WILD SALMON POLICY RAPID STATUS ASSESSMENT {#app:fourth-appendix}

See supporting document: "LIMIT REFERENCE POINTS AND WILD SALMON POLICY RAPID STATUS SUMMARY: YUKON CHINOOK and PORCUPINE CHINOOK STOCK MANAGEMENT UNITS" 

# COMPUTING ENVIRONMENT {#app:fifth-appendix}  

This document aims to be transparent and reproducible. If you made it this far in and are still paying attention you deserve a gold star. All data and code to reproduce the analysis in the report, and generate it, is available in [GitHub repository](https://github.com/Pacific-salmon-assess/yukon-CK-ResDoc/tree/main?tab=readme-ov-file#yukon-ck-resdoc).   

A document describing model diagnostics and some additional figures can be found within the repository at [`SR-diagnostics.html`](https://pacific-salmon-assess.github.io/yukon-CK-ResDoc/analysis/R/SR-diagnostics.html). 
R packages (and dependencies therein) necessary to recreate our analysis and documentation are: 
```{r env}
suppressWarnings({suppressMessages({
library(here)
library(tidyverse)
library(ggpubr)
library(ggcorrplot)
library(ggsidekick)
library(csasdown)
library(kableExtra)
library(rstan)
library(gsl)
library(cowplot)
library(viridis)
})})
pkgs <- sort(c("here", "tidyverse", "csasdown", "kableExtra", "rstan", "gsl", "cowplot", "viridis","ggpubr","ggcorrplot"))

pkg_table <- devtools::session_info()$packages %>%
  dplyr::filter(package %in% pkgs) %>%
  dplyr::select(package, loadedversion, date) %>%
  dplyr::rename(Package = package, Version = loadedversion, Date = date) 

rownames(pkg_table) <- NULL
csasdown::csas_table(pkg_table)
```
